org-bookmark-heading.el

```diff
@@ -130,6 +133,16 @@ heading.  Set org-id for heading if necessary."
 			    (handler . ,handler)
 			    (front-context-string . ,front-context-string)))))
 
+(defun org-bookmark-heading-default-name (filename heading)
+  "Return the default bookmark name.
+
+FILENAME is the path to the file.
+
+Intended as the default value for `org-bookmark-heading-name-fn'."
+  (let ((display-filename (funcall org-bookmark-heading-filename-fn filename)))
```

Comment:
Since this is so simple, I think we can omit binding `display-filename` and just put the `funcall` in the `concat`.  :)